framework/src/main/java/org/checkerframework/framework/flow/CFAbstractValue.java

```diff
@@ -48,12 +51,25 @@
  */
 public abstract class CFAbstractValue<V extends CFAbstractValue<V>> implements AbstractValue<V> {
 
+    /** Formats annotations without their package names. */
+    protected static DefaultAnnotationFormatter defaultAnnotationFormatter =
```

Comment:
Any reason to not make it `final`?
I would probably still move this into `toStringSimple`, because that's the only place that uses this field and that method won't be used much. But I'm fine with a static field.